Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
Traceback (most recent call last):
  File "snm.py", line 5, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
    from snm_simulation import snmSimulation
  File "../snm_simulation.py", line 4, in <module>
    import PyFrensie.Geometry as Geometry
    import PyFrensie.Geometry as Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
ImportError: No module named PyFrensie.Geometry
    import PyFrensie.Geometry as Geometry
ImportError: No module named PyFrensie.Geometry
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[35335,1],1]
  Exit code:    1
--------------------------------------------------------------------------
